jq.c

```diff
@@ -277,6 +289,96 @@ jdbc_detach_jvm()
 	(*jvm)->DetachCurrentThread(jvm);
 }
 
+static void jdbc_get_jni_env(void)
+{
+	int			JVMEnvStat;
```

Comment:
[minor] I'm not sure what the code style of this project is like, but this line contains 3 tabs between type and variable name, and it seems a bit too many to me.

Reply:
Thanks! This declaration came from the original code, so I wonder too why the author put three tabs here. I agree that this is redundant. I've replaced them with a space in https://github.com/scalar-labs/jdbc_fdw/pull/1/commits/c09af609a1e58336ce95275ebf074e426bf96f00.

Reply:
@komamitsu I've pushed one commit that adds a log message showing the current max heapsize (got with `Runtime.maxMemory()`) of the existing JavaVM. https://github.com/scalar-labs/jdbc_fdw/pull/1/commits/1f7ef768caa1b02d12ed2f1c9da414ad49ab9bae Could you let me know what you think of it? If it is too complicated, I would revert it.

Reply:
Here is the sample log meessage
```
WARNING:  Java VM has already been created by another extension. The existing Java VM will be re-used. The max heapsize may be different from the setting value. The current max heapsize is 954728448 bytes
```